src/content.js

```diff
@@ -1,4 +1,31 @@
 // this is the code that "Runs" the extension.
 // This is seperate from the code that runs the UI inside the extension.
-
 const deliberatelybroke = true;
+
+const videoTags = $('meta[name=\'keywords\']').attr('content').toLowerCase();
+const videoTitle = $('meta[name=\'title\']').attr('content').toLowerCase();
+const videoDesc = $('meta[name=\'description\']').attr('content').toLowerCase();
+let isMusic;
+
+for (let i = 0; i < 3; i++) {
```

Comment:
I'd like to do this with a different way. How about we make this into a function, such as "isMusic() which doesn't accept any arguments. 

then for these cases, instead we should have something along the lines of 
`
if (videoTags.split('music').length!=1) {
        return true;
      }`
 and at the very end of the function, have the statement "return false".
 
the reason is that inside a function, "return" will also end execution, which makes it so that you don't need to execute the remaining code, saving processor power as you wished to do.

Reply:
I agree that all of this should be at least one (probably several) functions. There's a lot of factoring and refinement to be done here